File: Javascript_sol/Design/Leetcode_705.js
var MyHashSet = function() {
  this.hmap = {};
};

/** 
* @param {number} key
* @return {void}
*/
MyHashSet.prototype.add = function(key) {
  this.hmap[key] = true
};

/** 
* @param {number} key
* @return {void}
*/
MyHashSet.prototype.remove = function(key) {
  this.hmap[key] = false
};

/** 
* @param {number} key
* @return {boolean}
*/
MyHashSet.prototype.contains = function(key) {
  if (this.hmap[key] != undefined) {return this.hmap[key]};
  return false   
};

/** 
* Your MyHashSet object will be instantiated and called as such:
* var obj = new MyHashSet()
* obj.add(key)
* obj.remove(key)
* var param_3 = obj.contains(key)
*/

File: Python_sol/Design/Leetcode_705.py
class MyHashSet:

    def __init__(self):
        self.hmap = {}
        

    def add(self, key: int) -> None:
        self.hmap[key] = True

    def remove(self, key: int) -> None:
        self.hmap.pop(key,0)


    def contains(self, key: int) -> bool:
        return self.hmap.get(key,0)
        


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)
###